apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -88,6 +89,36 @@ public string CredentialFile
             set => this[CredentialFileKeyword] = value;
         }
 
+        /// <summary>
+        /// Option to change between the default handling of null database values (return <see cref="DBNull.Value"/>) or
```

Comment:
I believe `<see cref="DBNull.Value"/>` will result in the hyperlink text being just "Value" in the docs. You may want to use `<see cref="DBNull.Value">DBNull.Value</see>` here instead.

Reply:
Done.